src/components/GasOptions/GasOptions.component.js

```diff
@@ -0,0 +1,213 @@
+    }
+    this.onFeeUpdated();
+  }
+
+  handleGasLimitChange(e) {
+    const { value } = e.target;
+    // allow an empty input to represent 0
+    if (value === '') {
+      this.setState({ gasLimitInput: '', gasLimit: 0 });
+      this.onFeeUpdated();
+    }
+
+    // only allow whole numbers
+    if (!gasLimitRegex.test(value)) return;
+
+    // don't allow infeasible amount of gas
+    // (gas limit per block almost never exeeds 10 million as of Aug 2018  )
+    const ONE_HUNDRED_MILLION = 100000000;
+    if (value > ONE_HUNDRED_MILLION) return;
+
+    this.setState({ gasLimitInput: value, gasLimit: parseInt(value, 10) });
+    this.onFeeUpdated();
+  }
+
+  handleOptionChange(type) {
+    const { gasStatistics } = this.props;
+    if (type === 'manual' || !gasStatistics) {
+      const { defaultGasLimit, defaultGasPrice } = this.props;
+      this.setState({
+        gasPriceGweiInput: defaultGasPrice.toString(),
+        gasPriceGwei: defaultGasPrice,
+        gasLimitInput: defaultGasLimit.toString(),
+        gasLimit: defaultGasLimit,
+        option: 'manual',
+      });
+    } else {
+      const suggestedGasPrice = gasStatistics[type] / 10;
+      this.setState({
+        gasPriceGweiInput: suggestedGasPrice.toString(),
+        gasPriceGwei: suggestedGasPrice,
+        option: type,
+      });
+    }
+    this.onFeeUpdated();
+  }
+
+  render() {
+    const { gasStatistics, defaultGasLimit, defaultGasPrice, intl } = this.props;
+    const { formatMessage } = intl;
+    const { gasLimitInput, gasPriceGweiInput, option } = this.state;
+    const disableInputs = option !== 'manual';
+    let gasOptions = [{ type: 'manual', name: 'Manual' }];
+    if (gasStatistics) {
+      gasOptions = [
+        { type: 'fast', name: `Fast <${Math.ceil(gasStatistics.fastWait)}min` },
+        { type: 'average', name: `Average <${Math.ceil(gasStatistics.avgWait)}min` },
+        { type: 'safeLow', name: `Slow <${Math.ceil(gasStatistics.safeLowWait)}min` },
+        { type: 'manual', name: 'Manual' },
+      ];
+    }
+    return (
+      <Collapse bordered={false} defaultActiveKey={['2']}>
+        <Panel
+          header={<AdvancedSettingsHeader>{formatMessage({ id: 'advanced_settings' })}</AdvancedSettingsHeader>}
+          key="1"
+        >
+          <FormItem label={<FormItemLabel>{formatMessage({ id: 'gas_options' })}</FormItemLabel>} colon={false}>
+            <Select
+              className="gas-options"
+              defaultValue={option}
+              onSelect={this.handleOptionChange}
+              style={{ paddingLeft: '0.5rem' }}
+            >
+              {
+                gasOptions.map((opt) => (
+                  <Option value={opt.type} key={opt.type}>
+                    {opt.name}
+                  </Option>
+                ))
+              }
+            </Select>
+          </FormItem>
+          <FormItem label={<FormItemLabel>{formatMessage({ id: 'gas_price' })}</FormItemLabel>} colon={false}>
```

Comment:
What do you think about hiding the "Gas Price" and "Gas Limit" inputs except when the option is manual?

Reply:
That way, we wouldn't need the 'Advanced Settings' dropdown anymore. 

Newb users would be able to change the fee without first clicking 'Advanced settings', and advanced users would be saved 1 extra click if they want to customise price and limit